### PR TITLE
Initial commits

### PR DESCRIPTION
**Pull request by kelvinqian00**

### CODE REVIEW

File: linter.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Linting with clj-kondo
```
Comment (milt) [inline linter.yml]: Why two of these?

Comment (kelvinqian00) [reply linter.yml]: Forgot to use `git rm` in addition to `git add`.

Comment (milt) [reply linter.yml]: me too, evertim

File: .github/workflows/linter.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout calling repository
+        uses: actions/checkout@v3
+
+      - name: Setup clj-kondo # No need to install Java for just clj-kondo
+        uses: DeLaGuardo/setup-clojure@9.5
+        with:
+          clj-kondo: ${{ inputs.clj-kondo-version }}
```
Comment (milt) [inline .github/workflows/linter.yml:29]: neat! Didn't know this was an option

**Review by milt — APPROVED** [review]
Looks good, let's lint!

File: .github/workflows/linter.yml (hunk 2 above)
Comment (kelvinqian00) [reply .github/workflows/linter.yml:29]: Yes this was one of my motivations of creating this separate workflow, to decouple clj-kondo installation from our usual Java-Clojure-Node setup.

**Pull request merged**